src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -225,7 +225,16 @@ fun processPathResolveVariants(lookup: ImplLookup, path: RsPath, isCompletion: B
     val parent = path.context
     val ns = when (parent) {
         is RsPath, is RsTypeElement, is RsTraitRef, is RsStructLiteral -> TYPES
-        is RsUseSpeck -> if (parent.isStarImport) TYPES_N_MACROS else TYPES_N_VALUES_N_MACROS
+        is RsUseSpeck -> when {
+            // use foo::bar::{self, baz};
+            //     ~~~~~~~~
+            // use foo::bar::*;
+            //     ~~~~~~~~~~~
```

Comment:
These is some inconsistency with underscore. `parent` is always a full use speck (with either group or start), so let's underscore either `foo::bar` or a full node.